### ISSUE TITLE
Administrator Interface Maintenance

### ISSUE THREAD

**Issue by zjanda** [raaamonnn/CS1D-European-Vacation-Project#5]
As an administrator,
I want to be able maintain the software by being able:
1. to add new European cities and their corresponding food items by having your program read 
    from the input file given to the class
2. to change the prices of the traditional food items
3. to add new traditional food items
4. to delete traditional food items

Size: 7

Estimate: Sprint 3

Definition of Done:
Done is when I can, without errors, add cities and their corresponding food items, change the prices of said food, add new traditional food items, and delete traditional food items.

List of Assumptions:
The admin ui needs to be implemented. The European cities class needs to be created as well as any classes related to food items. 
TLDR: European cities class, food class, admin UI.

Task:
Write code for classes and complete admin login. Implement features for admin page.

Test:
Add new cities and food items, change price of a food item, add new food item, and delete a food item. "Output" results to see if they're correct, whatever "output" means in this context.
